frontend/src/component/feature/FeatureView/FeatureVariants/FeatureEnvironmentVariants/EnvironmentVariantsCard/EnvironmentVariantsCard.tsx

```diff
@@ -127,13 +132,16 @@ export const EnvironmentVariantsCard = ({
                             onEditVariant={onEditVariant}
                             onDeleteVariant={onDeleteVariant}
                         />
-                        <Button
+                        <PermissionButton
+                            permission={permission}
+                            projectId={projectId}
+                            environmentId={environment.name}
                             onClick={onAddVariant}
                             variant="text"
                             startIcon={<Add />}
                         >
                             add variant
-                        </Button>
+                        </PermissionButton>
```

Comment:
This previously wasn't a permission button, which meant that you could click this, add a variant and get back a very sad error message if you didn't actually have permission to edit the variant

Reply:
Makes sense! I think you can clean up the previous Button import from MUI.

Reply:
Nice! Good call, removed